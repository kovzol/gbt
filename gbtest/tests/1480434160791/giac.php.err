// Using locale /usr/share/locale/
// C
// /usr/share/locale/
// giac
// UTF-8
// Maximum number of parallel threads 8
Help file /usr/share/giac/doc/local/aide_cas not found
Added 0 synonyms

// Groebner basis computation time 0.001613 Memory 0.02334M
// Time 0
0.03user 0.01system 0:00.05elapsed 94%CPU (0avgtext+0avgdata 25196maxresident)k
0inputs+24outputs (0major+1577minor)pagefaults 0swaps
