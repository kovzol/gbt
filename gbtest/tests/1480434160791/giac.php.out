Welcome to giac readline interface
(c) 2001,2016 B. Parisse & others
Homepage http://www-fourier.ujf-grenoble.fr/~parisse/giac.html
Released under the GPL license 3.0 or above
See http://www.gnu.org for license details
May contain BSD licensed software parts (lapack, atlas, tinymt)
-------------------------------------------------
Press CTRL and D simultaneously to finish session
Type ?commandname for help
0>> gbasis([-1*v9,-1*v7,-1+v10,v8,-1+v5,v3,v6,v4,v25^2*v24^2+-1*v26^2*v23^2,-<v10,v8,-1+v5,v3,v6,v4,v25^2*v24^2+-1*v26^2*v23^2,-1                         *v18+v16+v5+-1*v3,-1*v17+<2*v24^2+-1*v26^2*v23^2,-1*v18+v16+v5+-1*v3,-1*v17+v                         15+-1*v6+v4,v19*v18+-1*v2<v18+v16+v5+-1*v3,-1*v17+v15+-1*v6+v4,v19*v18+-1*v20                         *v17+-1*v19*v16+v17*v16+v<5+-1*v6+v4,v19*v18+-1*v20*v17+-1*v19*v16+v17*v16+v2                         0*v15+-1*v18*v15,-1*v6+v4<v17+-1*v19*v16+v17*v16+v20*v15+-1*v18*v15,-1*v6+v4+                         -1*v5*v4+v6*v3,-1*v19*v6+<*v15+-1*v18*v15,-1*v6+v4+-1*v5*v4+v6*v3,-1*v19*v6+v                         20*v5+v19*v4+-1*v5*v4+-1*<1*v5*v4+v6*v3,-1*v19*v6+v20*v5+v19*v4+-1*v5*v4+-1*v                         20*v3+v6*v3,v24^2+-1*v16^<0*v5+v19*v4+-1*v5*v4+-1*v20*v3+v6*v3,v24^2+-1*v16^2                         +-1*v15^2+2*v16*v14+-1*v1<0*v3+v6*v3,v24^2+-1*v16^2+-1*v15^2+2*v16*v14+-1*v14                         ^2+2*v15*v13+-1*v13^2,-1+<-1*v15^2+2*v16*v14+-1*v14^2+2*v15*v13+-1*v13^2,-1+v                         25^2+-1*v20^2+2*v19+-1*v1<2+2*v15*v13+-1*v13^2,-1+v25^2+-1*v20^2+2*v19+-1*v19                         ^2,-1*v5*v4+v6*v3+v5*v2+-<5^2+-1*v20^2+2*v19+-1*v19^2,-1*v5*v4+v6*v3+v5*v2+-1                         *v3*v2+-1*v6*v1+v4*v1,v23<2,-1*v5*v4+v6*v3+v5*v2+-1*v3*v2+-1*v6*v1+v4*v1,v23^                         2+-1*v16^2+-1*v15^2+2*v16<v3*v2+-1*v6*v1+v4*v1,v23^2+-1*v16^2+-1*v15^2+2*v16*                         v2+-1*v2^2+2*v15*v1+-1*v1<+-1*v16^2+-1*v15^2+2*v16*v2+-1*v2^2+2*v15*v1+-1*v1^                         2,-1*v9*v8+v10*v7+v9*v2+-<2+-1*v2^2+2*v15*v1+-1*v1^2,-1*v9*v8+v10*v7+v9*v2+-1                         *v7*v2+-1*v10*v1+v8*v1,v2<,-1*v9*v8+v10*v7+v9*v2+-1*v7*v2+-1*v10*v1+v8*v1,v26                         ^2+-1*v20^2+-1*v19^2+2*v2<v7*v2+-1*v10*v1+v8*v1,v26^2+-1*v20^2+-1*v19^2+2*v20                         *v2+-1*v2^2+2*v19*v1+-1*v<2+-1*v20^2+-1*v19^2+2*v20*v2+-1*v2^2+2*v19*v1+-1*v1                         ^2,1+-1*v16^2+-1*v15^2+2*<v2+-1*v2^2+2*v19*v1+-1*v1^2,1+-1*v16^2+-1*v15^2+2*v                         16*v2+-2*v1+2*v15*v1,-1*v<2,1+-1*v16^2+-1*v15^2+2*v16*v2+-2*v1+2*v15*v1,-1*v1                         5*v14+v16*v13+v15*v2+-1*v<6*v2+-2*v1+2*v15*v1,-1*v15*v14+v16*v13+v15*v2+-1*v1                         3*v2+-1*v16*v1+v14*v1],[v<*v14+v16*v13+v15*v2+-1*v13*v2+-1*v16*v1+v14*v1],[v2                         6,v24,v25,v23,v20,v18,v19<*v2+-1*v16*v1+v14*v1],[v26,v24,v25,v23,v20,v18,v19,                         v16,v17,v15,v10,v9,v8,v7,<,v24,v25,v23,v20,v18,v19,v16,v17,v15,v10,v9,v8,v7,v                         6,v5,v4,v3,v2,v1,v13,v14]<16,v17,v15,v10,v9,v8,v7,v6,v5,v4,v3,v2,v1,v13,v14],                         revlex)
[v13^2*v14^4+v14^6-4*v13^4-2*v13^2*v14^2-2*v14^4-8*v13^3-3*v13^2+v14^2,2*v15*v14^4-v13*v14^4+v14^4+4*v13^3+8*v15*v14^2-2*v13*v14^2+8*v15*v13-4*v13^2+8*v16*v14-6*v14^2+6*v15-13*v13-3,2*v15*v13*v14^2+v14^4+6*v15*v14^2-2*v13*v14^2+2*v15*v13-4*v13^2+4*v16*v14-4*v14^2+2*v15-6*v13-1,2*v16*v14^3-v14^4-2*v15*v14^2-4*v15*v13+4*v13^2-2*v16*v14-2*v15+4*v13+1,2*v15^2*v13+v15*v14^2+2*v15^2+2*v16*v14-v14^2+v15-2*v13-1,2*v15*v13^2+2*v15*v14^2+4*v15*v13-2*v13^2+2*v16*v14-v14^2+2*v15-4*v13-1,2*v16*v15*v14-v15*v14^2-2*v15^2-2*v16*v14+v14^2-v15+1,2*v15^2*v14-2*v15*v13*v14-v16*v14^2+2*v16*v15-4*v15*v14+2*v13*v14-v16+2*v14,v26^2-v15^2,v24^2+2*v15*v13-v13^2+2*v16*v14-v14^2-1,v25^2-v15^2+2*v15-1,v23^2-1,v16^2+v15^2-1,v16*v13-v15*v14,v20,v18-v16-1,v19-v15,v17-v15,v10-1,v9,v8,v7,v6,v5-1,v4,v3,v2,v1]
1>> 